code/game/machinery/adv_med.dm

```diff
@@ -179,7 +179,7 @@
 	if(!ishuman(connected.occupant))
 		to_chat(user, SPAN_WARNING("This device can only scan compatible lifeforms."))
 		return
-	if(!usr.stat_check(STAT_BIO, STAT_LEVEL_ADEPT))
+	if(!usr.stats?.getPerk(PERK_MEDICAL_EXPERT) && !usr.stat_check(STAT_BIO, STAT_LEVEL_ADEPT) && !usr.stat_check(STAT_COG, 50)) //Are we missing the perk AND to low on bio? Needs bio 25 so cog 50 to bypass
```

Comment:
Have you tried this as CRO, Roboticist and Scientist to check if they can bypass the BIO check with their COG stats?